client/store/actions.js

```diff
@@ -0,0 +1,7 @@
+const TYPES = {
+  SET_HEROES: 'SET_HEROES',
+};
+
+const setHeroes = (heroes) => ({ type: TYPES.SET_HEROES, heroes });
+
+export { TYPES, setHeroes };
```

Comment:
I think it's good to have a separate file with types. What do you think?

Reply:
I dont mind. ill separate them.